h/feeds/atom.py

```diff
@@ -56,6 +58,8 @@ def feed_from_annotations(
     :rtype: dict
 
     """
+    annotations = [h.AnnotationHTMLPresenter(a) for a in annotations]
```

Comment:
Oh, I think I see why you've done this. It's because `.title` has moved to the presenter class. In which case this is easily solved by moving the logic sections of `title` back onto the model, leaving the `title` property on the presenter to be something like:

```python
@property
def title(self):
    return jinja2.escape(self.annotation.title)
```